ledger/test-common/test-certificates/BUILD.bazel

```diff
@@ -5,37 +5,138 @@ load("@os_info//:os_info.bzl", "is_windows")
 
 genrule(
     name = "test-certificates",
-    srcs = [":openssl-extensions.cnf"],
+    srcs = [
+        ":openssl.cnf.sample"
+    ],
     outs = [
+        "openssl.cnf",
+        "newcerts",
```

Comment:
You can't have a directory as an output. (Well, you kind of can, but it's not as reliable as yo might like.)

You need to zip it up or something. It may be easier to just zip all of this up and have a single output, but that's your call.

Reply:
Does this need to be an output at all? We don’t seem to use it.

Reply:
This directory is required for `openssl ca` command to work. I didn't found a way to create a temporary directory for `genrule` and this is a workaround for that. However, I agree that we don't use the directory actually in the code.

Reply:
What I was trying to say is that we only use it within the rule so we don’t need it as an output just as a temporary directory.

Reply:
Fixed :)